### DISCUSSION TITLE
Removing/keeping title_bar_box layout

### DISCUSSION THREAD

**Discussion by Keith94** [Samillion/ModernZ#217] [General]
What's the opinion on the `title_bar_box` style? IMO it looks out of place in comparison to the bottom controls. Is there still a need to keep it available as a toggleable option?

Not exactly a harm in keeping it there, but it doesn't fit the same style anymore.

**Comment by Samillion** [comment]
I completely agree. Honestly, the only reason I kept it when I was re-doing the project is a "just in case it's a high demand".

It's one of those things where I'm never sure if it's personal preference or an actual useful action to take.

Nevertheless, if it were up to me only, I'd remove it.

**Reply by Keith94** [reply]
Can always remove it and see if anyone complains 😆 Since it's locked behind an option, I kinda doubt most people even tinkered with it.
